dev-tools/packaging/packages.yml

```diff
@@ -988,6 +1053,56 @@ specs:
           '{{.BeatName}}{{.BinaryExt}}':
             source: ./build/golang-crossbuild/{{.BeatName}}-{{.GOOS}}-{{.Platform.Arch}}{{.BinaryExt}}
     #### wolfi-based ####
+    #### Slim docker images ####
+    - os: linux
+      arch: amd64
+      types: [ docker ]
+      spec:
+        <<: *docker_ubi_spec
```

Comment:
We should have both UBI and Wolfi, not either or. For both Slim and EDOT.

Wolfi has license restrictions but will have no OS CVEs because they are distroless.

UBI is OSS but will have OS CVEs in it.

AFAIK, we need both variants for every image we intend to be widely used.